test/gleam/os_test.gleam

```diff
@@ -13,3 +14,15 @@ pub fn env_test() {
   |> map.get("GLEAM_TEST")
   |> should.equal(Error(Nil))
 }
+
+pub fn system_time_test() {
+  let june_12_2020 = 1591966971
+  {os.system_time(os.Second) > june_12_2020}
```

Comment:
The formatter strips these brackets, which case the tests to fail.

p.s. These tests seem silly but they do at least check a bit of the wiring to the underlying module. also they should be good till mid afternoon 6th Apri in the year 52417

Reply:
> they should be good till mid afternoon 6th Apri in the year 52417

😂 

> The formatter strips these brackets, which case the tests to fail.

Oh dear, I thought we fixed that. I'll open an issue.